server/pxf-api/src/main/java/org/greenplum/pxf/api/io/DataType.java

```diff
@@ -73,6 +71,17 @@
         }
     }
 
+    private static final Map<Integer, Integer> notText = new HashMap<>();
```

Comment:
Another option would be to use a set 
```suggestion
    private static final Set<Integer> notText = new HashSet<>();
```